lib/reporters/xunit.js

```diff
@@ -162,10 +162,15 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === STATE_FAILED) {
     var err = test.err;
+
+    if (Base.showDiff(err)) {
+      Base.stringifyDiffObjs(err);
+    }
```

Comment:
`err` has already been stringified in the `EVENT_TEST_FAIL` handler. Is their a reason you want to call it a second time?
I guess you want to make pass your test, but this is a bad way to achieve this.

Reply:
@juergba You're absolutely right, turns out I've been barking up the wrong tree because my test was wrong, thus misunderstanding what was happening. Very silly of me to not notice. :sweat_smile:  I'll revert my previous export of stringifyDiffObjs.